src/ui/Assets/Styles/patterns/_course-parts.scss

```diff
@@ -7,9 +7,27 @@
     padding-top: govuk-spacing(6);
     position: relative;
 
+    &--borderless {
+      border-top: none;
+    }
+
     &:first-of-type {
       margin-top: 0;
     }
+
+    .govuk-summary-list {
+      &__value {
```

Comment:
Wonder if this should be something like `__value--truncated` – in the future we will want some of these rows to wrap to new lines.

Reply:
Done